pages/filmEmpfehlung/index.js

```diff
@@ -0,0 +1,12 @@
+// This site is for showing a random movie, bassed on the passed movie Array
+
+import { movieData } from "../../assets/data/movieData";
+import RandomMoviePicker from "../../components/RandomMoviePicker/RandomMoviePicker";
+
+export default function randomMovie() {
+  return (
+    <>
+      <RandomMoviePicker movieArray={movieData} />
+    </>
```

Comment:
```suggestion
      <RandomMoviePicker movieArray={movieData} />
```
You only need fractions if you have two parent elements in the return statement :)

Reply:
done.